crates/apollo-router/Cargo.toml

```
@@ -10,16 +10,17 @@ name = "router"
 path = "src/main.rs"
 
 [features]
-default = ["otlp-tonic"]
-otlp-tonic = [
+default = ["otlp-grpc"]
+# activates OTLP/gRPC
+otlp-grpc = [
     "opentelemetry-otlp/tonic",
     "opentelemetry-otlp/tonic-build",
     "opentelemetry-otlp/prost",
     "tonic",
 ]
-otlp-grpcio = ["opentelemetry-otlp/grpc-sys", "opentelemetry-otlp/openssl"]
+# activates OTLP/HTTP
 otlp-http = ["opentelemetry-otlp/http-proto"]
```

Not a useful comment
```suggestion
otlp-http = ["opentelemetry-otlp/http-proto"]
```

---

right, I wrote those when otlp-tonic was still there, I can remove them now